src/components/MyCheckListItems/MyCheckListItems.tsx

```diff
@@ -0,0 +1,257 @@
+          // The current user has the role the checklist item is for
+          user.roles?.includes(item.Lead))
+    );
+
+    // Sort them in the order of the Lead, so that they can be grouped
+    myCheckListItems.sort((a, b) => a.Lead.localeCompare(b.Lead));
+
+    // Create the groups reference object that tells the ShimmeredDetailList how to render
+    groups = myCheckListItems.reduce(
+      (prevValue, curValue, curIndex) => {
+        if (prevValue.currentItem.key === curValue.Lead) {
+          // If the Lead of the Current Item is the same as the Previous, then increment the count
+          prevValue.currentItem.count++;
+          return prevValue;
+        } else {
+          // Otherwise, we have a new Lead, so create a currentItem object
+          prevValue.currentItem = {
+            key: curValue.Lead,
+            name: "Lead : " + curValue.Lead,
+            startIndex: curIndex,
+            count: 1,
+            level: 0,
+            isCollapsed: groupCollapsedState?.[curValue.Lead],
+          };
+          // Add a reference to this new object to the array of all groups
+          prevValue.all.push(prevValue.currentItem);
+          return prevValue;
+        }
+      },
+      // Provide an initial value to the reduce function, so that we begin evaluating on index 0 not 1
+      { currentItem: {} as IGroup, all: [] as IGroup[] }
+    );
+  }
+  /** Function to handle when the Complete button is clicked */
+  const completeCheckListItemClick = (itemId: number) => {
+    completeCheckListItem(itemId, {
+      onError: (error) => {
+        // If the mutation errored, then add an Error Notification
+        if (error instanceof ApiError) {
+          errorObj.addError(error.message);
+        } else {
+          errorObj.addError(
+            `An unknown error occured while trying to delete Checklist Item #${itemId}`
+          );
+        }
+      },
+    });
+  };
+
+  // Define columns for details list
+  const columns: IColumn[] = [
+    {
+      key: "item",
+      name: "Item",
+      fieldName: "Id",
+      minWidth: 40,
+      maxWidth: 40,
+      isResizable: false,
+    },
+    {
+      key: "title",
+      name: "Title",
+      fieldName: "Title",
+      minWidth: 100,
+      maxWidth: 200,
+      isResizable: true,
+    },
+    {
+      key: "complete",
+      name: "Complete",
+      minWidth: 100,
+      maxWidth: 200,
+      isResizable: true,
+      onRender: (item) => (
+        <Button
+          appearance="primary"
+          onClick={() => {
+            completeCheckListItemClick(item.Id);
+          }}
+        >
+          Complete
+        </Button>
+      ),
+    },
+    {
+      key: "empName",
+      name: "Employee Name",
+      fieldName: "request.empName",
+      minWidth: 100,
+      maxWidth: 200,
+      isResizable: true,
+      onRender: (item) => (
+        <Link to={"/item/" + item.ReqId}>{item.request.empName}</Link>
+      ),
```

Comment:
Agree -- Move to Title column.  When I did that, I decided to see if I could make the task open in the right side when it went to the request.  I decided to just push the link moving and not use this auto popup with the task because I realized it hides some of the Employee info under the panel.  And if the user was going to just mark the item as complete they would have done it from the listing not pull up the request itself.  I could see maybe some story with how to handle this, so the user can easily see the task the clicked on but also the Employee info.  May be a new view -- or maybe a new Panel that pops up on right side of MyCheckListItems that has the Employee info and the task info...